src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusPickupPointPlugin\DependencyInjection;
+
+use Setono\SyliusPickupPointPlugin\Provider\PostNordProvider;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+final class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('setono_sylius_pickup_point');
+        $rootNode
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->arrayNode('postnord')
```

Comment:
```suggestion
                ->arrayNode('post_nord')
```

This goes for all the other places using `postnord`